OpenUpSA/wazimap-ng
# Filter subcategory dropdown according to profile selected in profile indicator admin #257

## Description
It was working on staging but not on production so I created a new commit according to current code of stable branch 0.13.0
Also found a bug in authentication which prevented the super user to fetch api results for private profile.
Added SessionAuthentication in config to resolve authentication problem.

## Related Issue
#257 

## How to test it locally
Go into Profile Collection in admin panel and select profile.
Before selecting profile Subcategories dropdown should be empty.
It should send a ajax request to api for fetching subcategories for the profile.
Subcategories dropdown should have filtered options now 

## Changelog
* Added a js file
* Added get_form method in profile indicator admin
* Changed view of profile api `ProfileSubcategoriesList` to return all subcategories linked to a profile
* Added a new url
*  Added `SessionAuthentication` in config.

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
